oaknational/Oak-Web-Application
# Feat/spritesheet

## Description

- Adds a spritesheet so that svg graphics load with page, fixes #385 
- Adds [new icon set ](https://www.figma.com/file/lgns6S3DT4CbCqKQrEf0z4/Oak-Design-System-2022?node-id=2%3A14) fixes #363 
- Adds lesson element icons, fixes #364 
- Fixes button border graphic to be the right thickness

## How to test

1. Go to https://oak-web-application-storybook-puo759vph-oak-national-academy.vercel.app/?path=/docs/media-icon--icon
3. You should see the icons/graphics/lesson elements as below
4. Go to https://oak-web-application-storybook-puo759vph-oak-national-academy.vercel.app/?path=/docs/buttons-button--button-with-icon
5. You should see the button with icon as seen below

## Screenshots

![image](https://user-images.githubusercontent.com/12934669/183475446-271b41dd-30dc-40ad-9d3c-5b4405d63015.png)

![image](https://user-images.githubusercontent.com/12934669/183475760-004064a8-ad36-4b9c-9a7e-b13d14a37217.png)
